src/os/data/filedescriptor.js

```diff
@@ -173,25 +176,37 @@ os.data.FileDescriptor.prototype.getLayerOptions = function() {
 
 
 /**
- * Get the column mappings to apply to imported data.
- *
- * @return {Array.<os.im.mapping.IMapping>}
+ * @inheritDoc
  */
 os.data.FileDescriptor.prototype.getMappings = function() {
   return this.parserConfig['mappings'];
 };
 
 
 /**
- * Set the column mappings to apply to imported data.
- *
- * @param {Array.<os.im.mapping.IMapping>} value
+ * @inheritDoc
  */
 os.data.FileDescriptor.prototype.setMappings = function(value) {
   this.parserConfig['mappings'] = value;
 };
 
 
+/**
+ * @inheritDoc
+ */
+os.data.FileDescriptor.prototype.update = function() {
+  this.reimport();
```

Comment:
Can we reload the data without going through the whole reimport process? It felt confusing that I would need to go through the whole wizard again, and I imagine users will feel the same.